### ISSUE TITLE
Get China plugin translated to Chinese simplified characters?

### ISSUE THREAD

**Issue by langsmith** [mapbox/android-docs#1216]
Does it make sense to get https://docs.mapbox.com/android/plugins/overview/china/ translated to Simplified Chinese characters (简体)? 

Non-English-speaking Chinese developers might be on the hunt for Mapbox Android China info and would appreciate localized content regarding their region. 

Even if the developer manages to find https://www.mapbox.cn/tutorials/android/ , https://www.mapbox.cn/tutorials/android/  is different than the China Plugin.

_Additional idea:_ Should we add a link/blurb about https://www.mapbox.cn on the https://docs.mapbox.com/android/plugins/overview/china/ page?

cc @chriswu42

**Comment by chriswu42** [comment]
> Does it make sense to get https://docs.mapbox.com/android/plugins/overview/china/ translated to Simplified Chinese characters (简体)?

Yes we should, thanks for the input. @Jing-flyloveyin can you take lead on the translations? We can then ask @lloydsheng to update our cn doc page.

> Should we add a link/blurb about https://www.mapbox.cn on https://docs.mapbox.com/android/plugins/overview/china/ page

Yes we should point CN language docs over to mapbox.cn - @langsmith assuming this is maintained by the docs team?

**Comment by langsmith** [comment]
> Yes we should, thanks for the input. @Jing-flyloveyin can you take lead on the translations? We can then ask @lloydsheng to update our cn doc page.

Sounds good. Docs about the China plugin(s) would be good to have on a cn doc page.

This ticket was actually more about the suggestion to have the Simplified characters on our non-.cn docs page. To actually have characters on https://docs.mapbox.com/android/plugins/overview/china/ itself. Duplicate English/Characters on https://docs.mapbox.com/android/plugins/overview/china/. So we'd have the initial paragraph that starts with `Traditional map services are either blocked in China or...` and then have the equivalent paragraph in Simplified characters right below it. Same thing for all of the writing on that page.

My concern is that developers might come across https://docs.mapbox.com/android/plugins/overview/china/ before anything on mapbox.cn and think that https://docs.mapbox.com/android/plugins/overview/china/ is all that we've got for them. In reality, we've got everything that lives in the land of mapbox.cn. I guess this concern would be addressed by 👇 

> Yes we should point CN language docs over to mapbox.cn - @langsmith assuming this is maintained by the docs team?

Yes. If you're talking about adding a blurb/link on https://docs.mapbox.com/android/plugins/overview/china that points folks to mapbox.cn, that's something I can add. 

@mapbox/docs thoughts?

**Comment by chriswu42** [comment]
My preference is to keep everything on mapbox.com to English and mapbox.cn to Chinese (easier to maintain and control), if devs are looking at the China plugin docs on mapbox.com but want the Chinese version, we can link to the mapbox.cn respective pages in Chinese.
